Extension/src/Debugger/configurationProvider.ts

```diff
@@ -431,8 +431,11 @@ export class DebugConfigurationProvider implements vscode.DebugConfigurationProv
 
                 newConfig.name = configPrefix + compilerName + " " + this.buildAndDebugActiveFileStr();
                 newConfig.preLaunchTask = task.name;
+                const isCl: boolean = compilerName === "cl.exe";
                 if (newConfig.type === DebuggerType.cppdbg) {
                     newConfig.externalConsole = false;
+                } else if (isCl) {
+                    newConfig.console = "integratedTerminal";
                 } else {
                     newConfig.console = "externalTerminal";
```

Comment:
Is there a case where `newConfig.type` is `cppvsdbg` and not using `cl.exe`?

CppVsDbg is designed to be used only with MSVC.

I believe the change can be simplified to this one line being
```suggestion
                    newConfig.console = "integratedTerminal";
```

Reply:
I don't know of any case where cppvsdbg would be useable without cl.exe, so that seems good to me.